genai-perf/tests/test_model_config_measurement.py

```diff
@@ -0,0 +1,207 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
With so many additions, there is not a lot of unit testing.
Do you exercise every type of record?

Reply:
The record class has never had direct unit testing (it relies on indirect testing as it is used in a variety of unit testbenches). The MCM unit testing covers almost all of the base classes methods. And, the individual records will be exercised in unit tests once we port/create the code that handles the conversion of PA results to RCM/MCM.

Reply:
Good catch. There actually is some record type unit testing that I missed porting over. I'll add this on Monday.

Reply:
I've added the missing testing (and it also uncovered that I missed a file!). Please re-review when you get a chance.